streaming/base/format/base/writer.py

```diff
@@ -100,7 +100,8 @@ def __init__(self,
 
         # Validate keyword arguments
         invalid_kwargs = [
-            arg for arg in kwargs.keys() if arg not in ('progress_bar', 'max_workers', 'retry')
+            arg for arg in kwargs.keys()
+            if arg not in ('progress_bar', 'max_workers', 'retry', 'exist_ok')
```

Comment:
Can you please add the args comment for `exist_ok` at the start of the method? For example, something like below or in your own words
```
exist_ok (bool): If the local directory exists and is not empty, whether to overwrite
                the content or raise an error. `False` raises an error. `True` deletes the
                content and starts fresh. Defaults to `False`.
```

Also, do the same for all `writer.py` file in directory https://github.com/mosaicml/streaming/tree/main/streaming/base/format.